### ISSUE TITLE
Version 4.7.7 is still tagged as latest, instead of 4.8.0, making the updater not work

### ISSUE THREAD

**Issue by javiercastrode** [LinkStackOrg/LinkStack#774]
### LinkStack version

4.8.0

### Description

Version 4.7.7 is still tagged as latest, instead of 4.8.0, making the updater not work as it just updates to 4.7.7. It also makes the instance have a permanent notification asking the admin to update iit.

### Details about your system

GitHub
Any system

### How to reproduce

- To see that 4.7.7 is still tagged as latest: check the GitHub releases page
- To see the updater not working: try to update and check that it is still asking to update as well as the version remaining on 4.7.7

### Possible Solution

Change the tag

**Comment by JulianPrieber** [comment]
The updater works independent of the tags on GitHub

**Comment by javiercastrode** [comment]
Well, what I thought I saw was that it recognised that it has to update from 4.7.7 to 4.8.0 but it tried to download the new version from https://github.com/LinkStackOrg/LinkStack/releases/download/latest/update.zip or something similar, making it download the 4.7.7 release.
What I definitely could assure you is that when the updater finished it was still in 4.7.7, having tried it a couple of times.
However, now that 4.8.0 is marked as latest, the updater is still upgrading to 4.7.7 again, so the issue now appears to be something else. I will investigate further later and give an update.

**Comment by javiercastrode** [comment]
Well, what I thought I saw was that it recognised that it has to update from 4.7.7 to 4.8.0 but it tried to download the new version from https://github.com/LinkStackOrg/LinkStack/releases/download/latest/update.zip or something similar, making it download the 4.7.7 release.
What I definitely could assure you is that when the updater finished it was still in 4.7.7, having tried it a couple of times.
However, now that 4.8.0 is marked as latest, the updater is still upgrading to 4.7.7 again, so the issue now appears to be something else. I will investigate further later and give an update.

**Comment by javiercastrode** [comment]
I kept seeing the same behaviour, so I updated manually, which resulted in Laravel error "could not find driver: PRAGMA foreign_keys = ON;". After searching, it appeared to be solved by installing php-sqlite3 in its correct version (I had php-sqlite3 but not php8.2-sqlite3, the version appearing on the Laravel error page). After doing that, it worked, but it displayed a home page instead of my user, didn't recognize the login credentials and couldn't recover my password. Therefore, I reverted to the backup and tried the automatic updater again, successfully update.
After all, it appears to have been a question of updating dependencies. Will clear this up on a PR to documentation.
